vendor/vendor.json

```diff
@@ -1,6 +1,105 @@
 {
 	"comment": "",
 	"ignore": "test",
-	"package": [],
+	"package": [
```

Comment:
My understanding is that we want to ignore changes to this file, right? These are `govendor` specific changes.